resources/definitions/os_discovery/alaxala.yaml

```diff
@@ -0,0 +1,733 @@
+mib: AX-SMI-MIB:AX3630S:AX2430S:AX1230S:AX6300S:AX1240S:AX260A:AX2530S:AX3660S:AX4630S:AX7800S:AX7800R:AX2000R
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: ax3630sTemperatureStatusTable
```

Comment:
All oid, value and descriptions that use OIDs need to be in the format: `MIB-NAME::OID`